api/src/services/__tests__/competenciesService.ts

```diff
@@ -0,0 +1,19 @@
+import { listCompetencies } from '../competenciesService';
+import { mockQuery } from '../mockDb';
+
+describe('listCompetencies', () => {
+  it('should load a page that lists the competencies in the database with their titles and descriptions.', async () => {
+    mockQuery(
+      'select `id`, `label`, `description` from `competencies` limit ? offset ?',
+      [10, 10],
```

Comment:
What was your reason for using 10,10  for the limit and offset?

Reply:
1st 10 refers to the "code injection"/? after "limit" to limit the number of competencies requester at a time
2nd 10 refers to "code injection"/? after "offset" to consider competencies listed previously.